plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestingPinotCluster.java

```diff
@@ -66,7 +66,7 @@
 public class TestingPinotCluster
         implements Closeable
 {
-    private static final String BASE_IMAGE = "apachepinot/pinot:0.8.0-jdk11";
+    private static final String BASE_IMAGE = "apachepinot/pinot:0.9.3-jdk11";
```

Comment:
Do we have something like a smoke test for older version still? Or do we again plan to drop support?

Reply:
cc @xiangfu0 - wdyt?

Reply:
pinot keeps compatible support for the previous minor version upgrade. So typically 0.10.0 clients should work with 0.9.3 and 0.10.0 pinot. 0.8.0 pinot is not guaranteed.

Reply:
Thanks.

@elonazoulay Can you add two subclasses to AbstractPinotIntegrationSmokeTest - one that tests 0.9 and other that tests 0.10.

That way we can maintain tests and know the range of versions that the connector works with.
A similar idea like we have in some JDBC connectors.

Reply:
Sure! Will do.